cs/test/comm/Epoxy/EpoxyProtocolTests.cs

```diff
@@ -19,33 +19,38 @@ class EpoxyProtocolTests
     {
         private const uint GoodRequestId = 1;
         private const uint GoodResponseId = 1;
+        private const string GoodService = "MyService";
         private const string GoodMethod = "ShaveYaks";
         private const ProtocolErrorCode MeaninglessErrorCode = ProtocolErrorCode.GENERIC_ERROR;
         private static readonly IMessage<Dummy> meaninglessPayload = new Message<Dummy>(new Dummy());
-        private static readonly IMessage meaninglessError = Message.FromError(new InternalServerError() { error_code = (int)ErrorCode.InternalServerError, message = "Meaningless message"});
+        private static readonly IMessage meaninglessError = Message.FromError(new InternalServerError() { error_code = (int)ErrorCode.INTERNAL_SERVER_ERROR, message = "Meaningless message"});
```

Comment:
The previous version had the parens as well. Can we do a separate pass to fix things like that up?